modules/ols-about-lightspeed-custom-resource-definition.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+
+// * about/ols-configuring-openshift-lightspeed.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ols-about-lightspeed-custom-resource-definition_{context}"]
+= About the Lightspeed Custom Resource
+
+When you install the {ols-long} Operator, the OLSConfig Custom Resource (CR) gets installed in the {ocp-product-title} cluster. You can create a cluster-scoped custom resource named `cluster` to configure {ols-long} with your Large Language Model provider.
```

Comment:
When you install the {ols-long} Operator,  the OLSConfig Custom Resource Definition (CRD) gets installed in the @rh-tokeefe  .

Reply:
done